contracts/child/IParentToken.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.24;
+//interface for parent contract of any child token
+
+
+interface IParentToken {
+  function beforeTransfer(address user) public returns(bool);
```

Comment:
Please add two more arguments -- `(address to, uint256 value)`

Reply:
added